catalog/transformer/catalog-transformer-csv-queryresponse/src/test/java/ddf/catalog/transformer/csv/CsvMetacardTransformerTest.java

```diff
@@ -89,6 +98,10 @@ public void testCsvMetacardTransformer() throws CatalogTransformerException, IOE
     List<String> attNames = Arrays.asList(attributes.get(0).split(","));
     List<String> attValues = Arrays.asList(attributes.get(1).split(","));
 
+    for (int i = 0; i < COLUMN_ORDER.size(); i++){
```

Comment:
Consider making `COLUMN_ORDER` a list instead, then this could be `assertThat(attNames, is(COLUMN_ORDER))`